src/renderer/screens/exchange/Swap2/Form/FormSelectors/ToRow.js

```diff
@@ -1,40 +1,74 @@
 // @flow
-import React from "react";
+import React, { useEffect } from "react";
 import { Trans } from "react-i18next";
 import { BigNumber } from "bignumber.js";
 import Box from "~/renderer/components/Box/Box";
 import InputCurrency from "~/renderer/components/InputCurrency";
 import SelectCurrency from "~/renderer/components/SelectCurrency";
 import { amountInputContainerProps, selectRowStylesMap } from "./utils";
 import { FormLabel } from "./FormLabel";
-import { listCryptoCurrencies } from "@ledgerhq/live-common/lib/currencies";
-import type { CryptoCurrency } from "@ledgerhq/live-common/lib/types/currencies";
+import { toSelector } from "~/renderer/actions/swap";
+import { useSelector } from "react-redux";
+import { useSelectableCurrencies } from "~/renderer/screens/exchange/Swap2/utils/shared/hooks";
+import type { SwappableAccountType, toAccountType } from "./FormInputs";
+import { getAccountCurrency, getAccountUnit } from "@ledgerhq/live-common/lib/account";
+import type { useSelectableCurrenciesReturnType } from "~/renderer/screens/exchange/Swap2/utils/shared/hooks";
 
 type Props = {
-  toCurrency: ?CryptoCurrency,
-  setToCurrency: (?CryptoCurrency) => void,
+  fromAccount: ?SwappableAccountType,
```

Comment:
Did you wrap this type into its own thing for a specific reason? I'm so used to seeing the verbose
`Account | TokenAccount` everywhere that strikes me as odd.

Reply:
I wrapped this type to avoid explicitly call `Account | TokenAccount` everywhere but as I understand from your comments, this is the convention that the team choose so I will remove my wrapper and replace it.

Reply:
Done!